src/main/java/at/fhv/td/application/TicketController.java

```diff
@@ -0,0 +1,78 @@
+package at.fhv.td.application;
+
+import at.fhv.td.domain.Ticket;
+import at.fhv.td.domain.TicketAnswer;
+import at.fhv.td.domain.assembler.TicketAssembler;
+import at.fhv.td.persistence.broker.TicketBroker;
+import at.fhv.td.rmi.interfaces.ITicketDTO;
+
+import java.rmi.RemoteException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class TicketController {
+    private TicketAnswer _ticketAnswer;
+
+    TicketController() {
+        _ticketAnswer = new TicketAnswer();
+    }
+
+    public static TicketAnswer buyTicket(ITicketDTO ticketDto, Map<Integer, Integer[]> seatPlaceReservations) {
```

Comment:
Ist es nicht möglich hier in der Applikationsschicht keine DTO's bzw. Nichts mehr aus der RMI Schicht zu haben?
D.h. Alls DTO's in der RMI Schicht über Assembler umzuwandeln in Domainenobjekte?

Reply:
würde gehen, hab keine lust mehr nochmals zu ändern

Reply:
bzw vielleicht komm ich montags noch dazu